src/reporter/index.js

```diff
@@ -81,6 +82,10 @@ export default class Reporter {
         return find(this.reportQueue, i => i.test === testRun.test);
     }
 
+    _updateReportItem (reportItem, testRun) {
```

Comment:
This method is used only one time have unclear name.
Remove it and use content of it in-place  or change its signature to `static _addTestRunId(reportItem, testRun)`.

Reply:
I'd like to inject this code right in the `test-run-start` handler body